packages/fluentui/accessibility/test/behaviors/treeItemBehavior-test.tsx

```diff
@@ -50,11 +50,18 @@ describe('TreeItemBehavior', () => {
       expect(expectedResult.keyActions.root.performClick.keyCombinations[0].keyCode).toEqual(keyboardKey['Spacebar']);
     });
 
-    test(`click is executed with 'spacebar' or 'enter' key, when tree item is 'selectable' and tree item has subtree`, () => {
+    test(`selection is executed only with 'spacebar', when tree item is 'selectable'`, () => {
+      const expectedResult = treeItemBehavior({ selectable: true });
+      expect(expectedResult.keyActions.root.performSelection.keyCombinations).toHaveLength(1);
+      expect(expectedResult.keyActions.root.performSelection.keyCombinations[0].keyCode).toEqual(
+        keyboardKey['Spacebar'],
+      );
+    });
+
+    test(`click is executed with 'enter' key, when tree item is 'selectable' and tree item has subtree`, () => {
```

Comment:
Probably it's better to test this with screener or e2e tests

Reply:
Screener were also added

Reply:
Perfect